task1/KushalSosale.md

```diff
@@ -0,0 +1,140 @@
+~cat ./.hidden<br>
+***Learnt to read data from hidden files***<br>
+------------------
+**Level 4**   
+~ssh bandit4@bandit.labs.overthewire.org -p 2220<br>
+~ls<br>
+~cd inhere<br>
+~ls<br>
+~file ./-file*<br>
+~cat ./-file07<br>
+***Learnt to search for files with human readable data***<br>
+-----------------
+**Level 5**<br>
+~ssh bandit5@bandit.labs.overthewire.org -p 2220<br>
+~ls<br>
+~cd inhere<br>
+~ls<br>
+~find -type f -size 1033c ! -executable<br>
+~cd maybehere07<br>
+~cat ./.file2<br>
+***Learnt to search for files with a particular size***<br>
+------------------
+**Level 6**<br>
+~ssh bandit6@bandit.labs.overthewire.org -p 2220<br>
+~find / -user bandit7 -group bandit6 -size 33c 2>/dev/null<br>
+~cat /var/lib/dpkg/info/bandit7.password<br>
+***Learnt to search for files used and grouped by different users***<br>
+-------------------
+**Level 7**<br>
+~ssh bandit7@bandit.labs.overthewire.org -p 2220<br>
+~ls<br>
+~cat data.txt | grep millionth<br>
+***Learnt to search for text adjacent to a particular word***<br>
+---------------
+**Level 8**<br>
+~ssh bandit8@bandit.labs.overthewire.org -p 2220<br>
+~ls<br>
+~sort data.txt | uniq -u<br>
+***Learnt to search for unique data in file***<br>
+--------------
+**Level 9**<br>
+~ssh bandit9@bandit.labs.overthewire.org -p 2220<br>
+~cat data.txt | strings<br>
+~cat data.txt | strings | grep =<br>
+-----------
+**Level 10**<br>
+~ssh bandit10@bandit.labs.overthewire.org -p 2220<br>
+~cat data.txt | base64<br>
+~cat data.txt | base64 --decode<br>
+***Learnt to decode encoded data from txt file***<br>
+----------
+**Level 11**
+~ssh bandit11@bandit.labs.overthewire.org -p 2220<br>
+~ls<br>
+~cat data.txt | strings<br>
+~cat data.txt | tr "[n-za-mN-ZA-M]" "[a-zA-Z]"<br>
+***Learnt how to use 'tr' command and how to decode ROT cyphers***<br>
+------------
+**Level 12**<br>
+~mkdir /tmp/darkstar48<br>
+~cp data.txt /tmp/darkstar48<br>
+~cd /tmp/darkstar48<br>
+~file data.txt<br>
+~xxd -r data.txt > data<br>
+~file data<br>
+~mv data data.gz<br>
+~gunzip data.gz<br>
+~file data<br>
+~mv data data.bz2<br>
+~bzip2 -d data.bz2<br>
+~ls<br>
+~file data<br>
+~mv data data.gz<br>
+~gunzip data.gz<br>
+~ls<br>
+~file data<br>
+~mv data data.tar<br>
+~tar -xf data.tar<br>
+~ls<br>
+~file data5.bin<br>
+~mv data5.bin data5.tar<br>
+~tar -xf data5.tar<br>
+~ls<br>
+~file data6.bin<br>
+~mv data6.bin data6.bz2<br>
+~bzip2 -d data6.bz2<br>
+~ls<br>
+~file data6<br>
+~mv data6 data6.tar<br>
+~tar -xf data6.tar<br>
+~ls<br>
+~file data8.bin<br>
+~mv data8.bin data8.gz<br>
+~gunzip data8.gz<br>
+~ls<br>
+~file data8<br>
+~cat data8<br>
+***Learnt how to decode a Hexdump using 'xxd', how to unzip bzip2, gzip and tar files and how to use the 'mv' command to rename files***<br>
+----------
+**Level 13**<br>
+~ls<br>
+~ssh -i sshkey.private bandit14@localhost<br>
+~yes<br>
+~cd /etc/bandit_pass<br>
+~ls<br>
+~cat bandit1<br>
+***Learnt how to use SSH keys to connect.***<br>
+-------------------
+**Level 14**<br>

```

Comment:
It would be better if you have the same formatting for the last two levels as well.

Reply:
I don't know what seems to be the issue. I've done the exact same formatting.